proposed/http-handlers/request-handlers.md

```diff
@@ -83,17 +84,17 @@ use Psr\Http\Message\ResponseInterface;
 use Psr\Http\Message\ServerRequestInterface;
 
 /**
- * An HTTP request handler process a HTTP request and produces an HTTP response.
- * This interface defines the methods required to use the request handler.
+ * Handles a server request and produces a response
+ *
+ * An HTTP request handler process an HTTP request in order to produce an
+ * HTTP response.
  */
 interface RequestHandlerInterface
 {
     /**
-     * Handle the request and return a response.
+     * Handles a request and produces a response
```

Comment:
s/Handle/Handles/ :+1: 
Removing the period, :-1:

Reply:
The [phpdoc standard disagrees](https://github.com/php-fig/fig-standards/blob/master/proposed/phpdoc.md#51-summary):

> A Summary MUST end with either
> 
> * a full stop (.) followed by a line break
> * or two sequential line breaks.

Only a single-line summary ends with a period. If a description is included, it does not.